docs/deploy.rst

```diff
@@ -200,7 +195,7 @@ pushing to production.
   on homepage and article pages. Try to verify features in the newly pushed
   code. Check the `functional tests`_.
 
-* Announce in IRC that staging looks good, and you are pushing to production.
+* Announce in Matrix (#mdn) that staging looks good, and you are pushing to production.
```

Comment:
No, this is invalid but instead of removing, let's make a bit more true. 
```suggestion
* Announce in Slack (#mdn-dev) that staging looks good, and you are pushing to production.
```